index.php

```diff
@@ -172,5 +248,15 @@
     <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.3/jquery.min.js"></script>
     <script>window.jQuery || document.write('<script src="js/vendor/jquery.min.js"><\/script>')</script>
     <script src="js/bootstrap.min.js"></script>
+    <script type="text/javascript">
+      $(document).ready(function(){
+        $("#toggleForm").click(function(){
+          $("#jmsForm").toggle( "fast", function() {
+        });
+          $("#jmsToggle").toggle( "fast", function() {
+          });
+      });
```

Comment:
I think this can be as simple as (though I'm a bit rusty on jQuery):

```javascript
$(function() {
  const jmsForm = $('#jmsToggle');
  $('#toggleForm').click(function() {
    jmsForm.toggle('fast')
  });
});
```

Reply:
I'm switching 2 divs, hidding one and displaying other. But simplified it a bit, I'm super rusty in jQuery, so... :)